source/tyr/tyr/resources.py

```diff
@@ -138,6 +138,31 @@ def post(self, instance_name):
 
         return {'message': 'OK'}, 200
 
+    def delete(self, instance_name=None):
+        parser = reqparse.RequestParser()
+        parser.add_argument('state', type=str, required=False, case_sensitive=True, help='filter by job state')
+        args = parser.parse_args()
+
+        query = models.Job.query
+        if "state" in args and args["state"]:
+            query = query.filter_by(state=args["state"])
+        if instance_name:
+            query = query.join(models.Instance).filter(models.Instance.name == instance_name)
```

Comment:
Can't you just delete a job based on it's `id` ? Like if there is only one specific job that you want to be deleted ?

Reply:
BTW, GET /jobs/<job_id> isn't currently possible. Maybe it's worth adding it